### ISSUE TITLE
Set path to root when testing if cookies are enabled

### ISSUE THREAD

**Issue by stefanzweifel** [Alex1990/tiny-cookie#28]
When scanning a site which uses `tiny-cookie` through a service like https://www.onetrust.com/, we see an increased number in the reports of the `@key@`-cookie.

That's because the `@key@`-cookie is set to a relative path (eg. `/features/foo`, `/features/bar`) instead of a root path `/`.

https://github.com/Alex1990/tiny-cookie/blob/6fcecdb044465cf35fe0653b7ab6a495c697c3cc/src/index.js#L4-L19

Would you be open to a change, where the cookie path is updated to root (`/`)?
The changes would look like this:

```diff
- document.cookie = `${key}=${value}`;
+ document.cookie = `${key}=${value}; path=/`;
```

**Comment by Alex1990** [comment]
@stefanzweifel Thank you point it! It has been fixed in the latest version (v2.3.2).

**Comment by stefanzweifel** [comment]
@Alex1990 Thank you very much! ❤️
